playground/OpenAIEndToEnd/OpenAIEndToEnd.AppHost/Program.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var openai = builder.AddConnectionString("openai");
```

Comment:
Is it possible to use Parameters and ConnectionString together?

For example imagine I have:

```C#
var openaiKey = builder.AddParameter("OpenAIKey");

var openai = builder.AddConnectionString($"Key={openaiKey.Value}");
```

That way people wouldn't need to know about the connection string format. They would just stick the actual key into the parameter, and the above code would do the formatting for them.

Reply:
No that isn't possible at the moment. File an issue? There would need to be some consideration on how interpolation would work here because the "secretness" of one parameter would have to poison all the other parameters it was injected into (btw in this example you would want to use `AddParameter("OpenAIKey", secret: true)`. Only `AddConnectionString(...)` defaults to being secret.